bayesian/utility/v2/component_analyses.py

```diff
@@ -58,12 +67,28 @@ def normlize_packages(name: str, given_name: str,
 
 def unknown_package_flow(ecosystem: str, unknown_pkgs: Set[namedtuple]) -> bool:
```

Comment:
Though it is quite small, why can't this be considered as a util function? If it is, shall be shared with backbone as well.

Reply:
Yes, I am Okay to keep it in Utils, if team is fine with it, I can move it.

Reply:
@yzainee Please give your comment on it.

Reply:
As its not directly interacting with gremlin and all, i am ok with the idea of moving it to utils. please make sure you add a separate file for it with added test cases. utils has 95% test cov criteria